src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -139,6 +139,46 @@ public void testLegacyAccountEntityToAttributes() {
 
     }
 
+    @Test
+    public void testGetCopy_typicalData_shouldCreateDeepCopy() {
+        AccountAttributes account = createValidAccountAttributesObject();
+
+        AccountAttributes copy = account.getCopy();
+
+        assertNotSame(account, copy);
+        assertNotSame(account.studentProfile, copy.studentProfile);
+        assertFalse(account.isInstructor);
+
+        assertEquals(account.googleId, copy.googleId);
+        assertEquals(account.name, copy.name);
+        assertEquals(account.institute, copy.institute);
+        assertEquals(account.email, copy.email);
+    }
+
+    @Test
+    public void testGetCopy_ifAllFieldsNull_shouldCreateDeepCopy() {
```

Comment:
Based on our special naming convention, we don't need `test` `if` `should` (they are implied). It can be `getCopy_allFieldsNull_createsDeepCopy()`